src/mame/bfm/bfm_swp.cpp

```diff
@@ -192,8 +187,8 @@ INPUT_PORTS_END
 
 void bfm_swp_state::machine_start()
 {
-	m_cpuregion = (uint32_t*)memregion( "maincpu" )->base();
-	m_mainram = make_unique_clear<uint32_t[]>(0x10000);
+	m_cpuregion = (uint16_t*)memregion( "maincpu" )->base();
```

Comment:
You should really use a `required_region_ptr<uint16_t>` for this.

Reply:
OK, will change although I only changed it's size from 32 to 16. It was already there before.